modules/fts/pages/fts-demonstration-indexes.adoc

```diff
@@ -112,14 +113,15 @@ This page also provides a definition for the `geoIndex` definition, used in the
   "sourceParams": {},
   "planParams": {
     "maxPartitionsPerPIndex": 171,
+    "indexPartitions": 6,
     "numReplicas": 0
   },
   "uuid": ""
 }
 ----
 
-[#travel-sample-index-hotel-description]
-== Index Definition: travel-sample-index-hotel-description
+[#travel-sample-index-hotel-country-description]
+== Index Definition: travel-sample-index-hotel-country-description
```

Comment:
These changes will have an effect on the SDK docs, for example [Searching from the SDK](https://docs.couchbase.com/java-sdk/2.7/full-text-searching-with-sdk.html). Are there separate PRs to update all the SDK docs?

If none of the provided demonstration indexes are good for the new SEARCH() function example, it would be better to add an entirely new demonstration index for the SEARCH() function example, and leave travel-sample-index-hotel-description unchanged.

Reply:
Thanks for pointing that it out, I'll update the SDK documentation, there's a custom analzyer defined in that index def which isn't even picked up, so it's useless which i've dropped.

Reply:
On second thought, let me not update the sdk docs, but adapt the example i've used here.